PeerioTechnologies/peerio-desktop
# feat: new PeerUI SearchInput component

#### Relevant info and issue/PR links
https://app.clubhouse.io/peerio/story/17070/desktop-use-new-searchinput-component

#### Testing instructions
PeerUI has a new SearchInput component to replace the kinda hacky ways we were faking a search input in the app and copy-pasting tons of code. I also took this opportunity to do a bit of cleanup of some of the search JSX and CSS, e.g. removing the weird `chip-wrapper.scss` standalone file, un-nesting user search in a few places, and consolidating or removing old search input styles.

Functionality it should be the same as before, but there is a minor styling update to the input -- the corners are less rounded. They have a 4px border-radius, matching the previous Input update, instead of the fully semi-circle 24px border-radius.

Here's where SearchInput has been used:
* Contacts list
* NewContact search/invite
* Emoji search
* File picker (share-to-chat from Peerio files)
* Files view search bar
* Move file dialog
* Share dialog: in Files, click three-dots menu next to any file, click Share
* Share dialog: drag-and-drop multiple files into a chat, then click the "edit share" icon)
* User search in both new DM and new channel

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
